pc.c

```diff
@@ -216,6 +217,8 @@ bool SSL_ItronEnhanced_ExpandKey(
     memcpy(outptr + 1, xp, EC_POINT_PRIME256v1_X_SIZE);
     memcpy(outptr + 1 + EC_POINT_PRIME256v1_X_SIZE, yp, EC_POINT_PRIME256v1_X_SIZE);
 
+
+    mp_clear_multi(&param_p, &param_a, &param_b, &three, &alpha, &beta, &u, &y, &z, &xpi, &tmp1, &ypi, NULL);
```

Comment:
1
this code assumes that ExpandKey will 'always' be called after CompressKey which is incorrect.  This code need to be moved to another function within pc library so that caller can judge appropriate time to clear them.
2
split this long line